packages/sdk/cryptographic-utils/src/commentEncryption.ts

```diff
@@ -52,9 +52,6 @@ export function decryptData(data: Buffer, key: Buffer, sender: boolean): Buffer
     ? data.subarray(ECIES_SESSION_KEY_LEN, ECIES_SESSION_KEY_LEN * 2)
     : data.subarray(0, ECIES_SESSION_KEY_LEN)
   const sessionKey = ECIESDecrypt(u8(key), sessionKeyEncrypted)
-  if (sender) {
-    console.log(sessionKeyEncrypted)
-  }
```

Comment:
this was added when we upgrade to use noble before but i think it was accidentally left in. 


agree? @nicolasbrugneaux